### PR TITLE
Improve connection with local tunnel on README

### PR DESCRIPTION
**Pull request by alimtunc**
This is a simplest way to connect with a local tunnel using ngrok.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -628,17 +628,13 @@ The app should now be accessible at `http://localhost:3000`, and MongoDB at `mon
 Webrtc is working when on localhost, but if you want to test with another computer you need to have an HTTPS connection.  
 You will need to use a tunnel to expose you laptop over internet.
 
-We decided to use [localtunnel](https://github.com/localtunnel/localtunnel).  
-Once it's installed on an accessible server, setup env variable `LT_DOMAIN` without http(s) so just the domain.
+We decided to use [ngrok](https://ngrok.com/).
 
-After that, simply launch `ROOT_URL=https://lemverse-$(whoami).${LT_DOMAIN} meteor --settings settings-dev.json`.
+Once it's installed on an accessible server, on `settings-dev.json` replace `public.lp.website: http://localhost:3000` by `public.lp.website: NGROK_HTTPS_URL`
```
Comment (ramnes) [inline README.md]: We probably want to explain how to get that NGROK_HTTPS_URL

**Comment by xsyann** [comment]
You need to rebase interactive (i.e `rebase -i HEAD~5`) to remove the two unwanted commits and then rebase develop to your branch.

File: README.md (hunk 1 above)
Comment (alimtunc) [reply README.md]: I've updated the download url, they explain how to start a tunnel, I think it's enough

**Comment by alimtunc** [comment]
> You need to rebase interactive (i.e `rebase -i HEAD~5`) to remove the two unwanted commits and then rebase develop to your branch.

Just a `git rebase -i develop`do the work :)

File: README.md (hunk 1 above)
Comment (ramnes) [reply README.md]: my point is that if I read your instruction and don't pay enough attention, I could very well just copy paste `public.lp.website: NGROK_HTTPS_URL` without replacing the `NGROK_HTTPS_URL` part; it doesn't cost much to explain that you actually need to replace it

**Pull request closed without merging**